contrib/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM readthedocs/build:4.0rc1
```

Comment:
Shouldn't we inherit from `readthedocs/build:${label}`?

Reply:
yeah, not sure this is possible. if possible, we can make this dockerfile more complex. i'm fine without this as a first pass though.